src/Identity/test/Identity.Test/IdentityBuilderTest.cs

```diff
@@ -196,6 +196,26 @@ public void EnsureDefaultServices()
         Assert.IsType<UserManager<PocoUser>>(provider.GetRequiredService<UserManager<PocoUser>>());
     }
 
+    [Fact]
+    public void EnsureDefaultSignInManagerDependencies()
+    {
+        var services = new ServiceCollection()
+            .AddSingleton<IConfiguration>(new ConfigurationBuilder().Build());
+        services.AddLogging()
+            .AddIdentityCore<PocoUser>()
+            .AddRoles<PocoRole>()
+            .AddUserStore<NoopUserStore>()
+            .AddRoleStore<NoopRoleStore>()
+            .AddSignInManager<MySignInManager>();
```

Comment:
Can we add a variant in this same test class this test that just replaces the above with just since that's a different code path?

```suggestion
            .AddIdentity<PocoUser, PocoRole>();
```

Reply:
Sure.

Reply:
I'm afraid we need those stores at the very least. Otherwise, it complains about unresolvable services needed for `RoleManager` and `UserManager`. 

How about I add another variant of the test but based on `AddIdentity()` instead of `AddIdentityCore()` and with necessary stores?

Reply:
I've added additional test case for `AddIdentity()`.

Reply:
@halter73 Do you think it looks okay now?

Reply:
Bump.